File: Pi/src/Entity/Adresse.php
<?php

namespace App\Entity;
use Doctrine\Common\Collections\ArrayCollection;
use Doctrine\Common\Collections\Collection;
use Symfony\Component\Validator\Constraints as Assert;

use App\Repository\AdresseRepository;
use Doctrine\ORM\Mapping as ORM;


/**
 * @ORM\Entity(repositoryClass=AdresseRepository::class)
 */
class Adresse
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     */
    private $id;


    /**
     * @ORM\Column(type="string", length=255)
     *  @Assert\NotBlank(message="Name is required")
     */
    private $Nom;

    /**
     * @ORM\Column(type="string", length=255)
     *  @Assert\NotBlank(message="Last Name is required")
     */
    private $Prenom;

    /**
     * @ORM\Column(type="string", length=255)
     *  @Assert\NotBlank(message="Adress is required")
     */
    private $Adress;

    /**
     * @ORM\Column(type="string", length=255)
     *  @Assert\NotBlank(message="City is required")
     */
    private $City;

    /**
     * @ORM\Column(type="string", length=255)
     *  @Assert\NotBlank(message="Email is required")
     * @Assert\Email(message = "The email '{{ value }}' is not a valid email.")
     */
    private $Email;

    /**
     * @ORM\Column(type="integer")
     *  @Assert\Length(
     *     min = 8,
     *     max = 8,
     *     minMessage= "Le N°tel  est trop court",
     *     maxMessage="Le N°tel est trop long")
     * @Assert\NotBlank (message="Phone  is required")
     */
    private $Tel;

    /**
     * @ORM\OneToMany(targetEntity=Commande::class, mappedBy="adresse")
     */
    private $Commande;

    public function __construct()
    {
        $this->Commande = new ArrayCollection();
    }

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getNom(): ?string
    {
        return $this->Nom;
    }

    public function setNom(string $Nom): self
    {
        $this->Nom = $Nom;

        return $this;
    }

    public function getPrenom(): ?string
    {
        return $this->Prenom;
    }

    public function setPrenom(string $Prenom): self
    {
        $this->Prenom = $Prenom;

        return $this;
    }

    public function getAdress(): ?string
    {
        return $this->Adress;
    }

    public function setAdress(string $Adress): self
    {
        $this->Adress = $Adress;

        return $this;
    }

    public function getCity(): ?string
    {
        return $this->City;
    }

    public function setCity(string $City): self
    {
        $this->City = $City;

        return $this;
    }

    public function getEmail(): ?string
    {
        return $this->Email;
    }

    public function setEmail(string $Email): self
    {
        $this->Email = $Email;

        return $this;
    }

    public function getTel(): ?int
    {
        return $this->Tel;
    }

    public function setTel(int $Tel): self
    {
        $this->Tel = $Tel;

        return $this;
    }

    /**
     * @return Collection<int, Commande>
     */
    public function getCommande(): Collection
    {
        return $this->Commande;
    }

    public function addCommande(Commande $commande): self
    {
        if (!$this->Commande->contains($commande)) {
            $this->Commande[] = $commande;
            $commande->setAdresse($this);
        }

        return $this;
    }

    public function removeCommande(Commande $commande): self
    {
        if ($this->Commande->removeElement($commande)) {
            // set the owning side to null (unless already changed)
            if ($commande->getAdresse() === $this) {
                $commande->setAdresse(null);
            }
        }

        return $this;
    }
}

File: Pi java/src/Pi/entities/Adresse.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Pi.entities;

/**
 *
 * @author Aziz
 */
public class Adresse {
    int id;
    String Nom,Prenom,Adress,City,Email;
    int Tel;

    public Adresse() {
    }

    public Adresse(int id, String Nom, String Prenom, String Adress, String City, String Email, int Tel) {
        this.id = id;
        this.Nom = Nom;
        this.Prenom = Prenom;
        this.Adress = Adress;
        this.City = City;
        this.Email = Email;
        this.Tel = Tel;
    }

    public Adresse(String Nom, String Prenom, String Adress, String City, String Email, int Tel) {
        this.Nom = Nom;
        this.Prenom = Prenom;
        this.Adress = Adress;
        this.City = City;
        this.Email = Email;
        this.Tel = Tel;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNom() {
        return Nom;
    }

    public void setNom(String Nom) {
        this.Nom = Nom;
    }

    public String getPrenom() {
        return Prenom;
    }

    public void setPrenom(String Prenom) {
        this.Prenom = Prenom;
    }

    public String getAdress() {
        return Adress;
    }

    public void setAdress(String Adress) {
        this.Adress = Adress;
    }

    public String getCity() {
        return City;
    }

    public void setCity(String City) {
        this.City = City;
    }

    public String getEmail() {
        return Email;
    }

    public void setEmail(String Email) {
        this.Email = Email;
    }

    public int getTel() {
        return Tel;
    }

    public void setTel(int Tel) {
        this.Tel = Tel;
    }

    @Override
    public String toString() {
        return "Adresse{" + "id=" + id + ", Nom=" + Nom + ", Prenom=" + Prenom + ", Adress=" + Adress + ", City=" + City + ", Email=" + Email + ", Tel=" + Tel + '}';
    }
    
}
